nupic/regions/PCANode.py

```diff
@@ -435,36 +435,3 @@ def getOutputElementCount(self, name):
     else:
       raise Exception('Unknown output: ' + name)
 
-
-
-if __name__=='__main__':
-
-  from nupic.engine import *
-
-  rgen = numpy.random.RandomState(37)
-
-  inputSize = 8
-
-  net = Network()
-  sensor = net.addRegion('sensor', 'py.ImageSensor' ,
-          '{ width: %d, height: %d }' % (inputSize, inputSize))
```

Comment:
I agree that we should delete the main. Should we turn it into a test? Or put a simplified example in the module doc string?